frontend/src/scenes/pipeline/Pipeline.tsx

```diff
@@ -1,11 +1,35 @@
 import { SceneExport } from 'scenes/sceneTypes'
-import { pipelineLogic } from './pipelineLogic'
 import { PageHeader } from 'lib/components/PageHeader'
+import { humanFriendlyTabName, pipelineLogic, singularName } from './pipelineLogic'
+import { LemonTabs } from 'lib/lemon-ui/LemonTabs'
+import { useValues } from 'kea'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { urls } from 'scenes/urls'
+import { router } from 'kea-router'
+import { PipelineTabs } from '~/types'
 
 export function Pipeline(): JSX.Element {
+    const { currentTab } = useValues(pipelineLogic)
+
+    const singular = singularName(currentTab)
     return (
         <div className="pipeline-scene">
-            <PageHeader title="Pipeline" />
+            <PageHeader
+                title="Pipeline"
```

Comment:
Should this be labeled "CDP"?

Reply:
no acronyms :)

Reply:
If this is, in fact, the same product as what we have [on the website](https://posthog.com/docs/cdp), we should at least pluralize it so it partially matches: ~Customer Data~ Pipelines

Reply:
CDP = customer data platform (singular). I find pipelines weird as we don't have several atm.